libs/shared/routes/src/oeth/wrapOethWoeth.ts

```diff
@@ -69,11 +83,11 @@ const estimateGas: EstimateGas = async ({ config }, { amountIn }) => {
   try {
     if (publicClient) {
       gasEstimate = await publicClient.estimateContractGas({
-        address: tokens.mainnet.wOETH.address,
-        abi: tokens.mainnet.wOETH.abi,
+        address: wrappedTokens[tokenIn.chainId].address,
+        abi: wrappedTokens[tokenIn.chainId].abi,
         functionName: 'deposit',
-        args: [amountIn, whales.mainnet.OETH],
-        account: whales.mainnet.OETH,
+        args: [amountIn, whale[tokenIn.chainId]],
+        account: whale[tokenIn.chainId],
```

Comment:
We have designated whales for estimating gas?

Reply:
yea, just took the biggest token holders so that simulation doesn't fail